gef-ui/server/utils/helpers.js

```diff
@@ -18,7 +18,7 @@ const apiErrorHandler = ({ code, response }) => {
   }
   // Is validation error
   if (response.status === 422) {
```

Comment:
This might explain why you're only checking to 422 errors in other files. If this catches all other errors then ignore the comments above.

Reply:
Yeah, all errors are THROWN, except for 422 errors. Basically all errors will be caught in the catch method except for 422. Hope that makes sense

Reply:
Great